src/config.toml

```diff
@@ -31,30 +31,35 @@ identifier = "ds"
 url = "https://github.com/submariner-io/"
 weight = 10
 
+[[Languages.en.menu.shortcuts]]
+name = "<i class='fas fa-fw fa-slack'></i> Slack"
```

Comment:
class should use 'fab' and not 'fas'

Reply:
Added your commit to fix to this PR. Thanks again!